shell/devspace_start.sh

```diff
@@ -5,7 +5,7 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 # shellcheck source=./lib/bootstrap.sh
 source "$DIR/lib/bootstrap.sh"
 
-gh auth setup-git
+GH_NO_UPDATE_NOTIFIER=disabled gh auth setup-git
```

Comment:
Is this really valid? This is a double negative, I'd think it'd be `GH_UPDATE_NOTIFIER=disabled` or `GH_NO_UPDATE_NOTIFIER=true`

Reply:
Ah, I see, it's set to any value ([ref](https://cli.github.com/manual/gh_help_environment))

I'd suggest `true` instead so it's easier to read.

```suggestion
GH_NO_UPDATE_NOTIFIER=disabled gh auth setup-git
```

Reply:
I pulled the value out of issues implementing it. Those came up in google first. I can change to true.